lib/inferno/utils/middleware/request_logger.rb

```diff
@@ -35,7 +39,9 @@ def log_response(response, start_time, end_time, exception = nil)
           logger.info("#{status} in #{elapsed.in_milliseconds} ms")
           return unless body.present?
 
-          if body.length > 100
+          body = body.is_a?(Array) ? body.first : body
```

Comment:
Should this be `body.join` instead of `body.first`?  Or do you expect the full body to be in the first array element always?

Reply:
I can do that. I haven't seen any instances of multiple elements and I'm not sure what they would actually indicate.